facturacion_mexico/tests/test_setup_payment_terms.py

```
@@ -0,0 +1,69 @@
+	def setUpClass(cls):
+		super().setUpClass()
+		# Eliminar registros FM previos para probar creación desde cero
+		for name in _PT_NAMES:
+			if frappe.db.exists("Payment Terms Template", name):
+				frappe.delete_doc("Payment Terms Template", name, force=True)
+			if frappe.db.exists("Payment Term", name):
+				frappe.delete_doc("Payment Term", name, force=True)
+		frappe.db.commit()
+
+	@classmethod
+	def tearDownClass(cls):
```

Ruff (0.15.14)

[warning] Missing return type annotation for classmethod `tearDownClass`

Add return type annotation: `None`

(ANN206)